beacon-chain/blockchain/log.go

```diff
@@ -33,6 +34,12 @@ func logStateTransitionData(b interfaces.BeaconBlock) {
 	if len(b.Body().VoluntaryExits()) > 0 {
 		log = log.WithField("voluntaryExits", len(b.Body().VoluntaryExits()))
 	}
+	if b.Version() == version.Altair {
```

Comment:
This feels more like a debug log. In the working case, `syncAggParticipants` will always be 512

Reply:
Sorry I misunderstood. This is logging the actual count not the size. In this case, it is useful. I would rename it `syncAggBitsCount`